lib/resources/package.rb

```diff
@@ -44,8 +44,10 @@ def initialize(package_name = nil) # rubocop:disable Metrics/AbcSize
       elsif ['hpux'].include?(os[:family])
         @pkgman = HpuxPkg.new(inspec)
       else
-        return skip_resource 'The `package` resource is not supported on your OS yet.'
+        skip_resource 'The `package` resource is not supported on your OS yet.'
```

Comment:
You probably want to leave the `return` here - there's no reason to continue to evaluating missing requirements if the resource isn't supported on the target OS.